sonar-project.properties

```
@@ -5,9 +5,9 @@ sonar.projectName = libmav
 
 sonar.sources = include/,tests/
 
-sonar.exclusions = include/mav/rapidxml/,tests/doctest.h
+sonar.exclusions = include/mav/rapidxml/,tests/doctest.h,include/mav/picosha2/
 
-sonar.coverage.exclusions = tests/**/*,include/mav/rapidxml/
+sonar.coverage.exclusions = tests/**/*,include/mav/rapidxml/,include/mav/picosha2/
```

Pretty sure you need to add the `*`s here to actually exclude these.
```suggestion
sonar.coverage.exclusions = tests/**/*,include/mav/rapidxml/*,include/mav/picosha2/*
```